src/parser/transforms.js

```diff
@@ -476,9 +477,6 @@ module.exports.convertAst = function convertAst(result, preprocessedResult, visi
       }
     }
     if (node.type === 'GlimmerElementNode') {
-      if (node.tag === 'svg') {
-        path.context.inSvg = true;
```

Comment:
why remove this? i think we should check svg element only when we are in svg context?

Reply:
It's perfectly valid to create fragments like

```glimmer-js
// CustomPolygon.gjs
<template>
  <polygon path={{@path}} />
</template>
```

and use it elsewhere

```glimmer-js
<template>